app/views/admin/app_build_info.html.haml

```diff
@@ -0,0 +1,52 @@
+<h1>Build Information</h1>
```

Comment:
As this is using HAML, this can be written `%h1 Build Information`. The `h3` headers can be written similarly. Very minor - but no newline at the end of this file